### ISSUE TITLE
Back Shell?

### ISSUE THREAD

**Issue by ScapeGoat43** [philosiraptor/playstationzero#1]
Any chance that the back shell will be modeled?

**Comment by philosiraptor** [comment]
Yes, it has been modeled, it is not a replica , but it is close in dimensions, the model will be published when I get the original back from macho nacho so I can add port cutouts , expect it sometime next week

**Comment by ScapeGoat43** [comment]
many thanks for the response, and that's awesome that you guys was able to replicate the ps life controller, kudos to ya.

**Comment by Kasiin** [comment]
> Yes, it has been modeled, it is not a replica , but it is close in dimensions, the model will be published when I get the original back from macho nacho so I can add port cutouts , expect it sometime next week

really a good news, and one more question, would you share the design file of the panel?

**Comment by ScapeGoat43** [comment]
> Yes, it has been modeled, it is not a replica , but it is close in dimensions, the model will be published when I get the original back from macho nacho so I can add port cutouts , expect it sometime next week

hi phil, what ever happened to the rear shell?
